json-transformer/src/test/resources/logback.xml

```diff
@@ -3,7 +3,9 @@
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <!-- encoder default type is ch.qos.logback.classic.encoder.PatternLayoutEncoder-->
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} %level [%thread] %X{requestid} %logger{10} "%msg" %replace(%ex){'[\n]+', '\\n'}%nopex%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} %level [%thread] %X{requestid} %logger{10} "%msg" %replace(%ex){'[\n]+',
+                '\\n'}%nopex%n
```

Comment:
leave it as it was?

Reply:
sure. I was just indenting. No problem will leave it as it was 👍